src/WorkerChannel.ts

```diff
@@ -82,10 +82,13 @@ export class WorkerChannel implements IWorkerChannel {
    * @param msg gRPC message content
    */
   public workerInitRequest(requestId: string, msg: rpc.WorkerInitRequest) {
+    var capabilitiesDictionary = {};

```

Comment:
nit: `let` instead of `var` because [it's safer](https://www.educba.com/javascript-var-vs-let/)